application/common/components/phoneVerification/Sms.php

```diff
@@ -52,7 +52,7 @@ public function send($phoneNumber, $code)
         } catch (RequestException $e) {
             if ($e->hasResponse()) {
                 $response = $e->getResponse();
-                $body = $response->json();
+                $body = json_decode($response->getBody());
```

Comment:
You may need to add a 2nd argument to json_decode (set to `true`) so that the results will be an associative array rather than an object.

Typically when using json_decode, we do so like this:
```php
$body = json_decode($response->getBody(), true);
```